.travis.yml

```diff
@@ -1,24 +1,26 @@
 language: node_js
-node_js: 
-  - "stable" 
+node_js:
+- stable
 cache:
   directories:
-    - "node_modules"
+  - node_modules
 env:
   global:
-    - CC_TEST_REPORTER_ID=67adc75d217c19d11953e75dc6de8182a7ee803a55d94919192c3022b6741ee1    
+  - CC_TEST_REPORTER_ID=67adc75d217c19d11953e75dc6de8182a7ee803a55d94919192c3022b6741ee1
 before_script:
-  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
-  - chmod +x ./cc-test-reporter
-  - ./cc-test-reporter before-build
+- curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64
+  > ./cc-test-reporter
+- chmod +x ./cc-test-reporter
+- "./cc-test-reporter before-build"
 script:
- - npm test
+- npm test
 after_script:
-  - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT 
+- "./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT"
 deploy:
+  provider: heroku
   api_key:
-    secure: i55LFb5NZhR5SHh0OewcH62VLsg17jipAfMn/42eWYcmGfIern0zvpWnnLRyhg6Gyvn20+VwJIQtK06/jqKHP3stPFLQoNMj3+t7+NCuXjPAmceUkZCvnJXRlzBxp3jgfK5X6iot1ejzjewJEn++jqknYfbbw15WF7lQ2Aqgf2ni0KPix1bTjFAbumAY/J6qt9ZmnGgSFAJaqEbwCtf08TcO3GSFE17EuOYJtW++47uV2IyRZ03RQGPDrp/7TPa3ahc6aZfZczLxM4/yS/D90qvzW19qMWTwV5jAVRw6F2bR8e91ejGeAjBd5HbJHz0MMsotB8aqoWH0yhIp5dtXlMwHL2BXAsbpHYbLlTTPP9omZuJVoLxZu1nN14YPj1Dstp7KdBimBMaTgrYu+lXQlMwEyr3pw4aR4VWL+0wxmVzCEiCZPQYpm9//Jsqxn6wYClHJxdUOxcOpNRGw2Rq7OjBTnREp245n3MSuiYK/vzAIbR7x2vAs+b37FrGfjFbqeLdlRHZu8+D/cpou4zcVa3M4NEFYZZIgpuNHH7lQ+x+NIj0qH+TT/qG4z7iwzrsO2vdE8Mt+3OOESrwbsF86F1WHE7/ZvgR0VcVYjUr6bmuiV4/JQgGXjySrYYwwKJXdM3A7nEYMx8ou1j1fCIYe41R7AZ04Hpcz/kv8Lk/PNSc=
+    secure: 71e7c70f-935e-42ab-8223-3e45ce99e599
   app:
-    develop: savage-rangers-ah-staging
-    master : savage-rangers-ah
-after_success : npm run coverage
+    bg-fix-app-deployment-issue-#166494346: savage-rangers-ah-staging
```

Comment:
was it able to deploy with this branch here?

Reply:
yes it was

Reply:
If so it means the secure token is now working. try to put develop there again so that we're gona see if it is going to work when the pr will be merged

Reply:
now it's ready

Reply:
Let's just wait for @bdushimi to review and merge